src/birthder.js

```diff
@@ -56,46 +56,66 @@
         "Happy Birthday on your very special day, I hope that you don't die before you eat your cake."
     ];
 
-    // Selects random image URI from a list of images (returned by the search)
-    function select_tenor_image_url(response_objects) {
-        let top_anims = response_objects.results;
-        let rand_anim = top_anims[Math.floor(Math.random() * top_anims.length)];
-        return rand_anim.media[0].gif.url;
-    }
-
-    async function grab_tenor_image() {
-        let img_url, anon_id, response, tenor_key_response;
-        let tenor_key_url = "https://api.tenor.com/v1/anonid?key=" + TENOR_API_KEY;
-
-        response = await n_fetch(tenor_key_url);
-        tenor_key_response = await response.json();
-        // TODO process output for the wrong key, invalid input and so on
-        anon_id = tenor_key_response.anon_id;
-
-        let search_url = "https://api.tenor.com/v1/search?tag=" +
-            TENOR_SEARCH_TERM + "&key=" +
-            TENOR_API_KEY + "&limit=" +
-            TENOR_IMG_LIMIT + "&anon_id=" + anon_id;
-        // TODO process invalid inputs:
-        // check if response might be parsed as JSON,
-        // check if input dict contains required keys
-        response = await n_fetch(search_url);
-        img_url = select_tenor_image_url(await response.json());
-
-        return img_url;
-    }
-
     module.exports = function (robot) {
         let set_regex = /(birthday set) (?:@?([\w\d .\-_]+)\?*) ((0?[1-9]|[12][0-9]|3[01])\/(0?[1-9]|1[0-2])\/([\d]{4}))\b/i;
         let check_regex = /(birthdays on) ((0?[1-9]|[12][0-9]|3[01])\/(0?[1-9]|1[0-2])\/([\d]{4}))\b/i;
         let delete_regex = /(birthday delete) (?:@?([\w\d .\-_]+)\?*)\b/i;
 
-        // returns `true` if two dates have the same month and day of month
-        function check_dates_equal(dayA, dayB) {
-            return (dayA.month() === dayB.month()) && (dayA.date() === dayB.date());
+        /**
+         * Selects random image URI from a list of images (returned by the search)
```

Comment:
> Selects...

Third person, singular. However, in other cases imperative form of the verb is used. Choose one particular style and follow it.